Code/Framework/AzToolsFramework/AzToolsFramework/ComponentModes/BaseShapeViewportEdit.cpp

```diff
@@ -72,4 +82,20 @@ namespace AzToolsFramework
             AZ_ErrorOnce("BaseShapeViewportEdit", false, "No implementation provided for SetTranslationOffset");
         }
     }
+
+    void BaseShapeViewportEdit::BeginEditing()
+    {
+        if (m_beginEditing)
+        {
+            m_beginEditing();
+        }
+    }
+
+    void BaseShapeViewportEdit::FinishEditing()
```

Comment:
uber nit: This is the most pedantic thing ever so please do feel free to ignore. One thing I've been trying to do is pair `begin/end` and `start/finish` and not mix the two. I really don't think it matters in practice but I feel those are natural opposites. Make of that what you will 😅

Reply:
Well now I know the perfect way to troll you :p. I'll think about it in the next change.

Reply:
Perfect 😅